### PR TITLE
Added note about versioning to the README.

### PR DESCRIPTION
**Pull request by ConnorDoyle**

### CODE REVIEW

**Comment by ConnorDoyle** [comment]
Ping @air -- PTAL, added note to README as previously discussed offline.

File: README.md [hunk 1]
```diff
@@ -153,6 +155,26 @@ _Sample `command.json`._
...
+  "version": "0.1.0-alpha"
+}
+```
+_Sample `repo/meta/version.json`._
+
+This version is updated with any change to the required file content
+(typically validated using JSON schema) or expected file organization in the
+`repo` directory.
+
+_NOTE: The current version is `0.1.0-alpha` to facilitate rapid
```
Comment (air) [inline README.md]: Might be worth taking out this part, on the basis that we have to update the version in 2 places at the moment (risk of forgetting one).

**Comment by jsancio** [comment]
Ping. @ConnorDoyle let's resolve's @air comments so we can merge this.

**Comment by kensipe** [comment]
@ConnorDoyle is this ready to go?  I'm just cleaning up the universe :)

**Comment by ConnorDoyle** [comment]
@kensipe needs a rebase, feel free to take this over.

**Comment by kensipe** [comment]
@ConnorDoyle thx!  I'll rebase and merge today

**Comment by kensipe** [comment]
dang it!  This was merge into master... I'll fix it.  It should have been rebased to version-1.x

**Pull request merged**